log4j-core/src/main/java/org/apache/logging/log4j/core/layout/GelfLayout.java

```diff
@@ -237,11 +241,11 @@ public B setAdditionalFields(KeyValuePair[] additionalFields) {
     @Deprecated
     public GelfLayout(final String host, final KeyValuePair[] additionalFields, final CompressionType compressionType,
                       final int compressionThreshold, final boolean includeStacktrace) {
-        this(null, host, additionalFields, compressionType, compressionThreshold, includeStacktrace, true);
+        this(null, host, additionalFields, compressionType, compressionThreshold, includeStacktrace, true, false);
```

Comment:
Please do not change this method since that will break binary compatibility. Users will have to use builder to use the new feature.